lms/djangoapps/mobile_api/video_outlines/views.py

```diff
@@ -30,51 +30,48 @@ class VideoSummaryList(generics.ListAPIView):
         Get a list of all videos in the specified course. You can use the
         video_url value to access the video file.
 
-    **Example request**:
+    **Example Request**
 
         GET /api/mobile/v0.5/video_outlines/courses/{organization}/{course_number}/{course_run}
 
     **Response Values**
 
-        An array of videos in the course. For each video:
-
-            * section_url: The URL to the first page of the section that
-              contains the video in the Learning Management System.
-
-            * path: An array containing category, name, and id values specifying the
-              complete path the the video in the courseware hierarchy. The
-              following categories values are included: "chapter", "sequential",
-              and "vertical". The name value is the display name for that object.
-
-            * unit_url: The URL to the unit contains the video in the Learning
-              Management System.
+        If the request is successful, the request returns an HTTP 200 "OK"
+        response along with an array of videos in the course. The array
+        includes the following information for each video.
 
-            * named_path: An array consisting of the display names of the
+            * named_path: An array that consists of the display names of the
               courseware objects in the path to the video.
+            * path: An array that specifies the complete path to the video in
+              the courseware hierarchy. The array contains the following
+              values.
 
-            * summary:  An array of data about the video that includes:
+                * category: The type of division in the course outline.
+                  Possible values are "chapter", "sequential", and "vertical".
+                * name: The display name for the object.
+                * id: The The unique identifier for the video.
```

Comment:
@srpearce The "path" does include an "id".  You can see a real response here:
https://courses.edx.org/api/mobile/v0.5/video_outlines/courses/MITx/6.002_4x/3T2014

The example response must be outdated.